### PR TITLE
New App Musico using Spotify Api

### PR DESCRIPTION
**Pull request by leader282**
I have made a new app called 'Musico' using HTML, CSS and Vanilla JS. All guidelines are followed by me according to my current knowledge. It would be great if it could be reviewed and merged with the Hactoberfest tag.

### CODE REVIEW

**Comment by Tamanna-S** [comment]
add .gitignore in your repo and use templates or something for removing the ds files and secure your api key.

File: advance-projects/leader282-Musico/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const resultBox = searchInput.querySelector(".resultBox");
+const icon = searchInput.querySelector(".icon");
+let linkTag = searchInput.querySelector("a");
+let webLink;
+
+// if user press any key and release
+input.onkeyup = (e)=>{
+    let userData = e.target.value; //user enetered data
+    let emptyArray = [];
+    if(userData){
```
Comment (alicalimli) [inline advance-projects/leader282-Musico/index.js:13]: You can use Guard Clause here to avoid the code nesting "if (!userData) return"

Comment (leader282) [reply advance-projects/leader282-Musico/index.js:19]: Yeah, I found another way and implemented that

**Comment by leader282** [comment]
> add .gitignore in your repo and use templates or something for removing the ds files and secure your api key.

All done except the API part. Tried different methods but no help. Can you please guide me regarding the same?

**Comment by Tamanna-S** [comment]
https://stackoverflow.com/questions/44342276/how-to-push-code-to-github-hiding-the-api-keys

**Comment by leader282** [comment]
> https://stackoverflow.com/questions/44342276/how-to-push-code-to-github-hiding-the-api-keys

Yes, I tried .env thing but it works for node.js and not vanilla js I hope

**Comment by Tamanna-S** [comment]
you can do something like this https://github.com/alicalimli/movie-matrix-app/blob/main/src/scripts/config.js

create a file key.js, export key and import in you file and add the key.js in .gitignore

**Comment by leader282** [comment]
Please check once now

**Comment by Tamanna-S** [comment]
great!!

**Pull request merged**